romancal/tweakreg/tweakreg_step.py

```diff
@@ -0,0 +1,560 @@
+"""
+Roman pipeline step for image alignment.
+
+:Authors: Mihai Cara & Mairan Teodoro
+
+"""
+from os import path
+
+from astropy.table import Table
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from tweakwcs.imalign import align_wcs
+from tweakwcs.correctors import JWSTWCSCorrector
+from tweakwcs.matchutils import XYXYMatch
+
+# LOCAL
+from ..stpipe import RomanStep
+from roman_datamodels import datamodels
+from . import astrometric_utils as amutils
+from roman_datamodels.util import is_association
+
+
+def _oxford_or_str_join(str_list):
+    nelem = len(str_list)
+    if not nelem:
+        return 'N/A'
+    str_list = list(map(repr, str_list))
+    if nelem == 1:
+        return str_list
+    elif nelem == 2:
+        return str_list[0] + ' or ' + str_list[1]
+    else:
+        return ', '.join(map(repr, str_list[:-1])) + ', or ' + repr(str_list[-1])
+
+
+SINGLE_GROUP_REFCAT = ['GAIADR2', 'GAIADR1']
```

Comment:
Is there a reason we're limited to Gaia DR1 & DR2? I thought DR3 was out June 2022?

Reply:
I tested locally and saw no issues using DR3; added it to the list.